vizro-core/tests/unit/vizro/models/_components/test_container.py

```diff
@@ -23,10 +23,7 @@ def test_create_container_mandatory_only(self):
 
     def test_create_container_mandatory_and_optional(self):
         container = vm.Container(
-            title="Title",
-            components=[vm.Button(), vm.Button()],
-            id="my-id",
-            layout=vm.Layout(grid=[[0, 1]]),
+            title="Title", components=[vm.Button(), vm.Button()], id="my-id", layout=vm.Layout(grid=[[0, 1]])
```

Comment:
I'm always a fan of setting the `id` property as the first attribute, then `children` (or `components`), then the rest. It probably doesn't need to be part of this PR.

Reply:
I don't think we need to be super strict about this, as position shouldn't matter given that we use the keyword arguments. I can create a ticket though and we can consolidate this in another PR.